.mdformat.toml

```diff
@@ -0,0 +1 @@
+wrap = 80
```

Comment:
we use this tool only to enforce 80 character limit? 
how we chose 80? for historical reasons? If for some reason we are trying to be aligned with other more popular project I think the linux kernel increased this number to 100?

Reply:
if the tools has default somehow do we want to spell out to avoid that the tool decide to change defaults and then we are forced to do another huge patch like this that migrates all our doc to whatever new rule?

Reply:
basically we removed mdl_style.rb and i do not see in this PR any equivalent and i am curious to understand why because maybe that would have prevented also the need to touch all the doc if was already formatted per markdown rules

Reply:
> we use this tool only to enforce 80 character limit?

We use 80 for historical reasons (mdl enforced 80). I don't have opinions on what the exact number here should be, happy to change it to 100. 

> if the tools has default somehow do we want to spell out to avoid that the tool decide to change defaults and then we are forced to do another huge patch like this that migrates all our doc to whatever new rule?

The tool essentially just ensures that our markdown files are following the "GitHub Flavored Markdown" specification, so apart from the line length thing, it does not offer any configuration.

> basically we removed mdl_style.rb 

We used this file to mostly disable features of mdl. Since those are not present in mdformat in the first place, there's no need (or a possibility) to have an equivalent.

Reply:
> The tool essentially just ensures that our markdown files are following the "GitHub Flavored Markdown" specification, so apart from the line length thing, it does not offer any configuration.

I do not know the tool so i am not an expert of it. I quickly googled and find out that at least seems there are 3 options https://mdformat.readthedocs.io/en/stable/users/style.html so, i am interpret it correctly, my suggestion is to spell them out so it is clear which rules we are using and we are immune from potential changes dictated by that project to default values. Thanks.

Reply:
Added that link to the body of the first commit. mdformat does not allow any configuration for these, so we cannot spell them out anywhere.

Reply:
sorry wrong link, i meant https://mdformat.readthedocs.io/en/stable/users/configuration_file.html

Reply:
Ah, yeah, added those two options

Reply:
thanks for the change